src/regexmanager.cpp

```diff
@@ -222,18 +223,57 @@ void RegexManager::remove_last_regex(const std::string& location)
 	regexes.erase(it);
 }
 
-std::string RegexManager::extract_initial_marker(const std::string& str)
+std::string RegexManager::extract_outer_marker(const std::string str, const int index)
 {
+	regmatch_t pmatch;
+	regex_t regex;
+	// Non-zero number or non-angle bracket characters, enclosed in angle brackets
+	const char *regexpat = "<[^<>]+>";
+	std::string close = "</>";
+	std::string tmptag;
+	std::stack<std::string> tagstack;
+	int offset = 0;
+
 	if (str.length() == 0)
 		return "";
```

Comment:
Please apply the boyscout rule and put this line in curly braces. Same goes for an `if` below.

Reply:
Take only photographs, leave only curly braces. Done, Thanks